doc/btp_gap.txt

```diff
@@ -695,6 +684,36 @@ Commands and responses:
 
                 In case of an error, the error response will be returned.
 
+        Opcode 0x2b - Sent subrate request
+
+                Controller Index:       <controller id>
+                Command parameters:     Address_Type (1 octet)
+                                        Address (6 octets)
+                                        Subrate_Min (2 octets)
+                                        Subrate_Max (2 octets)
+                                        Conn_Latency (2 octets)
+                                        Continuation_Number (2 octets)
+                                        Supervision_timeout (2 octets)
+                Return Parameters:      <none>
+
+                This command is used to send a subrate request to a connected
```

Comment:
This should also generate event "Connection Subrate Update event" similar to how Connection Parameters Update is handled.